sglang/code-walk-through/readme.md

```diff
@@ -69,7 +69,7 @@ TokenizerManager runs in Server's main weprocess, handling request tokenization.
 
 ## Scheduler Receive Requests and Process Batches
 
-The Scheduler runs as Server’s subprocess, initialized via `run_scheduler_process` and executing its infinite event loop with `event_loop_normal` or `event_loop_overlap`.
+The Scheduler runs as Server’s subprocess, initialized via `run_scheduler_process` and executing its infinite event loop with `event_loop_normal` or `event_loop_overlap`. Details can be found in [scheduler.py](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/managers/scheduler.py).
```

Comment:
For all these kind of references, I would suggest embedding the link the in the first mention , e.g."The [Scheduler](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/managers/scheduler.py) runs as Server's subprocess..." to save space?

Reply:
Yes this was just to keep the style consistent but I agree this way is better, but it should be adjusted everywhere like this to have consistent style.
For example

TokenizerManager runs in Server's main weprocess, handling request tokenization. It is initialized in launch_server, with details in [tokenizer_manager.py](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/managers/tokenizer_manager.py).

-> 

[TokenizerManager](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/managers/tokenizer_manager.py) runs in Server's main weprocess, handling request tokenization. It is initialized in launch_server etc.

Reply:
Sure, let's make it consistent